### PR TITLE
add endpoint

### PR DESCRIPTION
**Pull request by supun-ebi**

### CODE REVIEW

File: support-data-rest/src/main/java/org/uniprot/api/support/data/statistics/controller/StatisticsController.java [hunk 1]
```diff
@@ -79,7 +79,44 @@ public StatisticsController(StatisticsService statisticsService) {
 
     @Operation(
             hidden = true,
-            summary = "Get release statistics by UniProt release name and statistics type.",
+            summary = "Get release statistics by UniProt release name.",
```
Comment (LeonardoGonzales) [inline support-data-rest/src/main/java/org/uniprot/api/support/data/statistics/controller/StatisticsController.java:82]: Can you please add summary as a constant??

File: support-data-rest/src/main/java/org/uniprot/api/support/data/statistics/controller/StatisticsController.java [hunk 2]
```diff
@@ -79,7 +79,44 @@ public StatisticsController(StatisticsService statisticsService) {
...
+        return ResponseEntity.ok()
+                .body(
+                        new StatisticsModuleStatisticsResult<>(
+                                statisticsService.findAllByVersionAndCategoryIn(
+                                        release, categories)));
+    }
+
+    @Operation(
+            hidden = true,
+            summary = "Get history by attribute and statistics type.",
```
Comment (LeonardoGonzales) [inline support-data-rest/src/main/java/org/uniprot/api/support/data/statistics/controller/StatisticsController.java:119]: Can you please add it as a constant as well?

File: support-data-rest/src/test/java/org/uniprot/api/support/data/statistics/controller/StatisticsControllerIT.java [hunk 3]
```diff
@@ -129,6 +188,39 @@ void getByReleaseAndTypeAndMultipleCategories() throws Exception {
...
+    void getByReleaseAndMultipleCategories() throws Exception {
+        this.mockMvc
+                .perform(
+                        get(
+                                "/statistics/releases/2021_03?categories=EUKARYOTA,TOP_ORGANISM"))
+                .andDo(log())
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.results.size()", is(2)))
+                .andExpect(jsonPath("$.results[0].categoryName", is("TOP_ORGANISM")))
+                .andExpect(jsonPath("$.results[0].searchField", is("sf Organism")))
```
Comment (LeonardoGonzales) [inline support-data-rest/src/test/java/org/uniprot/api/support/data/statistics/controller/StatisticsControllerIT.java:201]: What is the searchField attribute means?

Comment (supun-ebi) [reply support-data-rest/src/test/java/org/uniprot/api/support/data/statistics/controller/StatisticsControllerIT.java:201]: It's not a part of this task. 
There's a column called 'search_field' in statistic_category table (for some categories not all).

**Pull request closed without merging**